BUILD.gn

```diff
@@ -94,11 +95,33 @@ npm_action("atom_browserify_sandbox") {
     "-p",
     "tsconfig.electron.json",
     "]",
+    "--standalone",
+    "sandboxed_preload",
     "-o",
     rebase_path(outputs[0]),
   ]
 }
 
+js_wrap("atom_browserify_wrap") {
+  deps = [
+    ":atom_browserify_content_script",
+    ":atom_browserify_isolated",
+    ":atom_browserify_sandbox",
```

Comment:
consider breaking this into 3 separate targets with 1 input/output each. there's no particular reason all these need to be built together.